packages/cli/src/commands/mock.ts

```diff
@@ -1,43 +1,97 @@
 import { Command } from '@oclif/command';
+import { createLogger, logLevels } from '@stoplight/prism-core';
+import chalk from 'chalk';
+import * as cluster from 'cluster';
+import { LogDescriptor, Logger } from 'pino';
 import * as signale from 'signale';
-import { ARGS, FLAGS } from '../const/options';
+import * as split from 'split2';
+import { PassThrough, Readable } from 'stream';
+import { ARGS, FLAGS, LOG_COLOR_MAP } from '../const/options';
 import { createServer } from '../util/createServer';
 
+function logEnabledFeature(message: string) {
+  signale.star({
+    prefix: chalk.bgWhiteBright.black('[CLI]'),
+    message,
+  });
+}
+
 export default class Server extends Command {
   public static description = 'Start a mock server with the given spec file';
-  public static flags = { port: FLAGS.port, host: FLAGS.host, dynamic: FLAGS.dynamic };
+  public static flags = FLAGS;
   public static args = [ARGS.spec];
 
   public async run() {
-    const signaleInteractiveInstance = new signale.Signale({ interactive: true });
-
     const {
-      flags: { port, dynamic, host },
+      flags: { port, dynamic, host, multiprocess },
       args: { spec },
     } = this.parse(Server);
 
-    signaleInteractiveInstance.await('Starting Prism…');
-
-    if (dynamic) {
-      signale.star('Dynamic example generation enabled.');
-    }
+    if (cluster.isMaster) {
```

Comment:
This change has so many new terms in it. cluster, stream, multipurpose, pino, master, signale, there is so much not mocking command leaking into the mocking command. I fear this will confuse the heck out of anyone who works on it.

Could you maybe make a stiry with sone notes how to refactor this? It's not just about reuse for a 2nd command, its a problem for understanding this one command.

Reply:
Yeah ok, this is definitely confusing; let me try to come up with a better way to reorganise the thing.

Reply:
@philsturgeon I've teared out stuff as much as possible here. 72336ba

Reply:
It's still a monster, but it should be very well enclosed in two nice functions.